DOCS/interface-changes.rst

```diff
@@ -31,6 +31,7 @@ Interface changes
 ::
 
  --- mpv 0.41.0 ---
+    - add `--d3d11-output-mode` enable use specific output mode for creating the D3D11 swapchain.
```

Comment:
Don't edit this file directly, add dedicated file in interface-changes directory.